### PR TITLE
voting, majority & passed clarified (closes #82)

### LINKED ISSUE #82: Feature Request: Improved clarity of vote history
**Is your feature request related to a problem? Please describe.**
When referring to the vote history popup, it's easy at a glance to see the Majority number for a given vote and think that it's the vote total for that vote. As a separate column it stands out more than the number next to the "people" icon and names.

I'm honestly not sure if the "Majority" number as a stand-alone number is even that useful. I'd prefer a quick-glance checkmark or X to denote whether this vote *achieved* a majority - I rarely need to know after-the-fact what the majority threshold was for a particular vote.

**Describe the solution you'd like**
Some options:
- Move the majority column away from the "Hand up" column
- Make the total vote number more prominent as its own column or combine it with majority as "5 / 6" or "8 / 6" etc
- Add an easy-to-read "did this vote pass or not" indicator (check or x or similar)

**Additional context**
I may try to put together a PR for this :)
image

### PR DESCRIPTION
**Pull request by JulGvoz**
Tiny tweak, implements the 2nd and the 3rd option of the original issue.

### CODE REVIEW

File: src/components/modals/VoteHistoryModal.vue [hunk 1]
```diff
@@ -18,21 +18,24 @@
           <td>Nominator</td>
           <td>Nominee</td>
           <td>Type</td>
-          <td>Majority</td>
+          <td>Votes</td>
           <td><font-awesome-icon icon="hand-paper" /> Hand up</td>
+          <td>Passed</td>
```
Comment (bra1n) [inline src/components/modals/VoteHistoryModal.vue:23]: Maybe call it "Majority" instead? Whether or not it passed could be influenced by roles like Thief or Bureaucrat, which could make "Passed" misleading.

File: src/components/modals/VoteHistoryModal.vue [hunk 2]
```diff
@@ -18,21 +18,24 @@
...
         </tr>
       </thead>
       <tbody>
         <tr v-for="(vote, index) in session.voteHistory" :key="index">
           <td>{{ vote.nominator }}</td>
           <td>{{ vote.nominee }}</td>
           <td>{{ vote.type }}</td>
-          <td>{{ vote.majority }}</td>
+          <td><strong>{{ vote.votes.length }}</strong> / {{ vote.majority }}</td>
```
Comment (bra1n) [inline src/components/modals/VoteHistoryModal.vue:31]: I'd probably remove the `{{vote.majority}}` here...

File: src/components/modals/VoteHistoryModal.vue [hunk 3]
```diff
@@ -18,21 +18,24 @@
...
           <td>
-            {{ vote.votes.length }}
             <font-awesome-icon icon="user-friends" />
             {{ vote.votes.join(", ") }}
           </td>
+          <td>
+            <font-awesome-icon icon="check-square" v-if="vote.votes.length >= vote.majority" />
```
Comment (bra1n) [inline src/components/modals/VoteHistoryModal.vue:37]: ...and add it here in parenthesis.

```suggestion
            <template v-if="vote.votes.length >= vote.majority">
              <font-awesome-icon icon="check-square" /> ({{ vote.majority }})
            </template>
```

Comment (JulGvoz) [reply src/components/modals/VoteHistoryModal.vue:37]: Ah, I agree, that seems to be a more sensible spot to put it.

**Comment by bra1n** [comment]
I think it would be best to just show the majority where the nomination passed.
Also, maybe we could roll https://github.com/bra1n/townsquare/issues/81#issuecomment-754175396 into this PR as well? A simple HH:mm formatted timestamp should not be too much work, what do you think @JulGvoz?

**Comment by JulGvoz** [comment]
Unfortunately, I'm not really experienced with vue to be able to do that myself, though I agree that HH:mm format would work.

I still think that all votes should show the majority needed, as that also helps the players orient them selves in the nomination history.

**Comment by bra1n** [comment]
> Unfortunately, I'm not really experienced with vue to be able to do that myself, though I agree that HH:mm format would work.

No worries, I'll add that myself then. :-)
 
> I still think that all votes should show the majority needed, as that also helps the players orient them selves in the nomination history.

Fair enough, do you want to make this change?

> (also, this is my first PR, I don't really know how to commit the suggestions you did to the PR)

Just make a new commit to your fork's main branch, that will automatically show up in the PR here.

**Comment by JulGvoz** [comment]
image
This is how it currently looks like. I think it is clear enough.

**Comment by JulGvoz** [comment]
image

This is how bra1n's version looks like, I'll merge this one so we can merge the rest of the changes regarding time etc

**Comment by bra1n** [comment]
Awesome, thank you very much, @JulGvoz!

**Pull request merged**